BuildForSDG/FASTA
# add illustrations

## Description
Please include a summary of the change and relevant motivation and context. 
List any dependencies that are required for this change.

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #77 

## How Has This Been Tested?

1. Clone the ft-frontend-add-illustrations branch
2. make sure you have nodejs installed on your machine then run npm install
3. after installing all the dependencies run npm run dev
4. open your browser and navigate to localhost:300

## Screenshots (if applicable, else remove this line / section)


## Checklist:
